Apps/W1/HybridGP/app/src/pages/GPMigrationConfiguration.Page.al

```diff
@@ -293,6 +293,21 @@ page 4050 "GP Migration Configuration"
                             until GPCompanyAdditionalSettings.Next() = 0;
                     end;
                 }
+                field("Skip Posting Item Batches"; Rec."Skip Posting Item Batches")
+                {
+                    Caption = 'Item Batches';
+                    ToolTip = 'Specify whether to disable auto posting Item batches.';
+                    ApplicationArea = All;
+
+                    trigger OnValidate()
+                    begin
+                        if PrepSettingsForFieldUpdate() then
+                            repeat
+                                GPCompanyAdditionalSettings.Validate("Skip Posting Item Batches", Rec."Skip Posting Item Batches");
+                                GPCompanyAdditionalSettings.Modify();
+                            until GPCompanyAdditionalSettings.Next() = 0;
```

Comment:
We are missing FindSet() or read operation, it is odd to iterate through the set without finding it first. It can lead to unpredictable behavior.

Reply:
This is done in the PrepSettingsForFieldUpdate() procedure.